pages.pt_BR/linux/abroot.md

```diff
@@ -0,0 +1,33 @@
+# abroot
+
+> O utilitário ABRoot fornece total imutabiliodade e atomicidade ao transacionar entre 2 estados da partição raíz (A⟺B).
+> Isso também permite transações sob demanda via um shell transacional.
+> Mais informações: <https://github.com/Vanilla-OS/ABRoot>.
+
+- Saída do estado da partição raiz atual ou futuro:
+
+`sudo abroot get {{presente|futuro}}`
```

Comment:
I think the argument option shouldn't be translated
```suggestion
`sudo abroot get {{present|future}}`
```

Reply:
Yes, they should not if they are some sort of keywords for a command.

Reply:
I thought these words would be replaced by the argument as named on the user's system, so I have translated it.

Reply:
It would be nice if it worked so. :) (I guess it doesn't.)

Reply:
> I think the argument option shouldn't be translated

Yep, exactly.